### ISSUE TITLE
Dependency conflict with devbox

### ISSUE THREAD

**Issue by daviiiL** [MarceColl/zen-browser-flake#55]
Thank you for packaging zen! 

I have found a dependency collision with devbox tho. 

Error message attached and hope it helps!

```shell
 error: collision between `/nix/store/pd2r92zkyrq6d8yn3gmrsdxk95cvxaa5-devbox-0.10.7/bin/updater' and `/nix/store/s2bz22csj1a8yjh3n3i1h67s5zhrh1g2-zen-browser-1.0.1-a.6/bin/updater'
```

**Comment by adamdicarlo** [comment]
Here's the fix: https://github.com/jetify-com/devbox/pull/2426

**Comment by daviiiL** [comment]
> Here's the fix: [jetify-com/devbox#2426](https://github.com/jetify-com/devbox/pull/2426)

Thank you!
